apps/spotlight/src/layouts/Layout.astro

```diff
@@ -49,10 +49,12 @@ const context = getAppContext();
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
+    <script is:inline src={`${context.uswdsRoot}js/uswds-init.min.js`}></script>  
     <script src="../lib/initialize.ts"></script>
     <title>{title}</title>
   </head>
   <body id="top">
+    <script is:inline src={`${context.uswdsRoot}js/uswds.min.js`}></script>
```

Comment:
I see the script in here twice, could we remove one? I'm wondering also about `is:inline` - does that address a problem? Inlining it adds a bit of bloat, so would be nice to not inline if not necessary.

Reply:
My bad, I see one is the init script, and one is the complete bundle. Makes sense to inline the init script, but I think we can skip inlining on `uswds.min.js`?

Reply:
> My bad, I see one is the init script, and one is the complete bundle. Makes sense to inline the init script, but I think we can skip inlining on `uswds.min.js`?

I agree with this. I'd suggest adding `defer` to the `uswds.min.js` script tag.

Reply:
Hi guys. Regarding the uswds scripts, per the [documentation site for uswds](https://designsystem.digital.gov/documentation/developers/#using-uswds-css-and-javascript-in-your-project), both scripts need to be loaded, one in the head and one in the body. 

The reason why both scripts have the "is:inline" directive is because per the [astro documentation site](https://docs.astro.build/en/guides/client-side-scripts/#load-external-scripts), it's needed in order to load scripts that are not in the astro "src" folder. I tested this by attempting to load the scripts without the directive and they don't show up in the DOM at all.

@ethangardner I added the defer to the uswds.min.js script tag.